src/plugins/visualizations/public/saved_visualizations/saved_visualizations.ts

```diff
@@ -76,5 +76,9 @@ export function createSavedVisLoader(services: SavedObjectKibanaServicesWithVisu
     }
   }
   const SavedVis = createSavedVisClass(services);
-  return new SavedObjectLoaderVisualize(SavedVis, savedObjectsClient, services.chrome);
+  return new SavedObjectLoaderVisualize(
+    SavedVis,
+    savedObjectsClient,
+    services.chrome
+  ) as SavedObjectLoader & { findListItems: (search: string, size: number) => any };
```

Comment:
Is the casting necessary here too? In reading the code it seems it should "just work" (but haven't played with this file locally yet)

Reply:
same as above